rules-tests/Php80/Rector/Identical/StrStartsWithRector/config/polyfill_configured_rule.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+
+use Rector\Core\ValueObject\PhpVersion;
+use Rector\Core\ValueObject\PolyfillPackage;
+use Rector\Php80\Rector\Identical\StrStartsWithRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->rule(StrStartsWithRector::class);
+
+    $rectorConfig->phpVersion(PhpVersion::PHP_74);
+
+    $rectorConfig->polyfillPackages([PolyfillPackage::PHP_80]);
```

Comment:
It needs test if `rectorConfig->polyfillPackages()` is not enabled, to verify it skipped

Reply:
I created a PR to add test for it:

- https://github.com/rectorphp/rector-src/pull/5391